Remove unused parameter from function logger.setupFileLogger

Hi there,

Suggesting removal of an unused paramater in [mean/config/lib/logger.js](https://github.com/meanjs/mean/blob/c8cbcd326f4ab4253a090fbafc2d0af336471258/config/lib/logger.js)

Down in the function `logger.setupFileLogger = function setupFileLogger(options) {`

Notice the function parameter `options`?

- It isn't used anywhere within that function.
- At [the one place where logger.setupFileLogger is called](https://github.com/meanjs/mean/search?utf8=%E2%9C%93&q=setupFileLogger), no argument is passed.

Very minor issue, essentially just a tidy-up.

What does everyone think?

Setting no-unused-vars will hiccup quite a bit on issues so that's probably not the time to enable it:
```
[13:09:38] 
gulpfile.js
  10:3  error  'testConfig' is defined but never used  no-unused-vars

server.js
  7:5  error  'server' is defined but never used  no-unused-vars

config/config.js
  125:49  error  'assets' is defined but never used  no-unused-vars

config/env/production.js
  3:5  error  'fs' is defined but never used  no-unused-vars

config/lib/express.js
  172:54  error  'app' is defined but never used  no-unused-vars

config/lib/logger.js
  42:51  error  'options' is defined but never used  no-unused-vars

config/lib/mongoose.js
  23:7  error  '_this' is defined but never used  no-unused-vars

config/lib/seed.js
   7:3   error  'crypto' is defined but never used  no-unused-vars
  58:43  error  'reject' is defined but never used  no-unused-vars
  72:11  error  'User' is defined but never used    no-unused-vars

config/lib/socket.io.js
  81:70  error  'err' is defined but never used  no-unused-vars

modules/users/server/controllers/users/users.authentication.server.controller.js
  206:51  error  'next' is defined but never used  no-unused-vars

modules/core/client/config/core.client.route-filter.js
  14:68  error  'fromParams' is defined but never used  no-unused-vars

modules/core/client/config/core.client.routes.js
  23:55  error  '$location' is defined but never used  no-unused-vars

modules/core/client/controllers/home.client.controller.js
  9:9  error  'vm' is defined but never used  no-unused-vars

modules/core/client/directives/show-errors.client.directives.js
  36:11  error  'initCheck' is defined but never used  no-unused-vars

modules/users/client/directives/password-verify.client.directive.js
  20:11  error  'status' is defined but never used  no-unused-vars

modules/articles/client/controllers/admin/article.client.controller.js
  39:32  error  'res' is defined but never used  no-unused-vars

modules/users/client/controllers/settings/change-password.client.controller.js
  27:79  error  'response' is defined but never used  no-unused-vars

modules/articles/tests/server/admin.article.server.routes.tests.js
    3:5   error  'should' is defined but never used          no-unused-vars
   66:33  error  'signinRes' is defined but never used       no-unused-vars
   79:42  error  'articleSaveRes' is defined but never used  no-unused-vars
  111:33  error  'signinRes' is defined but never used       no-unused-vars
  118:13  error  'userId' is defined but never used          no-unused-vars
  161:33  error  'signinRes' is defined but never used       no-unused-vars
  168:13  error  'userId' is defined but never used          no-unused-vars
  188:33  error  'signinRes' is defined but never used       no-unused-vars
  195:13  error  'userId' is defined but never used          no-unused-vars
  230:9   error  'articleObj' is defined but never used      no-unused-vars
  235:33  error  'signinRes' is defined but never used       no-unused-vars
  242:13  error  'userId' is defined but never used          no-unused-vars
```

--- a/config/lib/logger.js
+++ b/config/lib/logger.js
@@ -37,9 +37,8 @@ logger.stream = {
 /**
  * Instantiate a winston's File transport for disk file logging
  *
- * @param logger a valid winston logger object
  */
-logger.setupFileLogger = function setupFileLogger(options) {
+logger.setupFileLogger = function setupFileLogger() {
 
   var fileLoggerTransport = this.getLogOptions();
   if (!fileLoggerTransport) {
@@ -72,13 +71,9 @@ logger.setupFileLogger = function setupFileLogger(options) {
  *
  * Returns a Winston object for logging with the File transport
  */
-logger.getLogOptions = function getLogOptions(configOptions) {
+logger.getLogOptions = function getLogOptions() {
 
   var _config = _.clone(config, true);
-  if (configOptions) {
-    _config = configOptions;
-  }
-
   var configFileLogger = _config.log.fileLogger;
 
   if (!_.has(_config, 'log.fileLogger.directoryPath') || !_.has(_config, 'log.fileLogger.fileName')) {
@@ -141,6 +136,6 @@ logger.getLogFormat = function getLogFormat() {
   return format;
 };
 
-logger.setupFileLogger({});
+logger.setupFileLogger();
 
 module.exports = logger;

